src/test/java/com/jcabi/http/mock/MkAnswerSimpleTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2011-2015, jcabi.com
```

Comment:
@mkordas  [ERROR] \src\test\java\com\jcabi\http\mock\MkAnswerSimpleTest.java[2]: Line does not match expected header line of ' * Copyright (c) 2011-2015, jcabi.com'. (HeaderCheck)